cmd/antrea-controller/controller.go

```diff
@@ -93,7 +93,7 @@ func run(o *Options) error {
 
 	var traceflowController *traceflow.Controller
 	if features.DefaultFeatureGate.Enabled(features.Traceflow) {
-		traceflowController = traceflow.NewTraceflowController(crdClient, traceflowInformer)
+		traceflowController = traceflow.NewTraceflowController(podInformer, crdClient, traceflowInformer)
```

Comment:
nit: put the `podInformer` parameter after the `crdClient`

Reply:
fixed